modules/internal/ChapelArray.chpl

```diff
@@ -1874,6 +1874,58 @@ module ChapelArray {
     pragma "no doc"
     proc indexOrder(i) return _value.dsiIndexOrder(_makeIndexTuple(rank, i));
 
+    /*
+       Returns the `ith` index in the domain counting from 0. 
```

Comment:
This ends up looking weird because of indentation in doc-string:

![Screen Shot 2021-01-20 at 2 46 11 PM](https://user-images.githubusercontent.com/3595449/105226659-4b9aee00-5b2e-11eb-844a-02455f30811c.png)

I suggest keeping all documentation indented to the same level to fix this.